src/Tribe/Service_Providers/Body_Classes.php

```diff
@@ -20,6 +20,7 @@ class Body_Classes extends \tad_DI52_ServiceProvider {
 	 */
 	public function register() {
 		tribe_singleton( Body_Class_Object::class, Body_Class_Object::class );
+		tribe_singleton( 'tribe_body_classes_provider', $this );
```

Comment:
Check the naming standards for common: https://moderntribe.github.io/products-engineering/docs/plugins/namespacing/